docs/src/Button.doc.js

```diff
@@ -21,24 +21,33 @@ You are able to specify the color, type, and width of buttons to change their ap
 card(
   <PropTable
     props={[
+      {
+        name: 'accessibilityControls',
+        type: 'string',
+        description:
+          'Identifies the element whose contents or presence are controlled by the current element. Populates aria-controls.',
+        href: 'accessibility-disclosure',
+      },
       {
         name: 'accessibilityExpanded',
         type: 'boolean',
         description:
-          'Use this property on elements that can expand to reveal additional information',
+          'Use this property on elements that can expand to reveal additional information. Populates aria-expanded.',
+        href: 'accessibility-disclosure',
       },
       {
         name: 'accessibilityHaspopup',
         type: 'boolean',
         description:
-          'Indicates that the element has a popup context menu or sub-level menu.',
+          'Indicates that the element has a popup context menu or sub-level menu. Populates aria-haspopup.',
+        href: 'accessibility-popup',
       },
       {
         name: 'accessibilityLabel',
         type: 'string',
         description:
-          'String that clients such as VoiceOver will read to describe the element. Always localize the label.',
-        href: 'accessibilityLabel',
+          'String that clients such as VoiceOver will read to describe the element. Always localize the label. Populates aria-label.',
```

Comment:
`Always localize the label`.

This might be an internal requirement, not a Gestalt API requirement (open source).

Reply:
Good point. We removed that since it's already an internal expectation that any text should be localized.